.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: julia
+os:
+  - linux
+  - osx
+julia:
+  - 1.0
+  - 1.3
+  - nightly
+
+notifications:
+  email:
+    recipients:
+      - nightly-rse@invenia.ca
+    on_success: never
+    on_failure: always
+    if: type = cron
```

Comment:
@oxinabox might need to set this bit up?

Reply:
Doing now

Reply:
done

Reply:
done

Reply:
thanks